OPT: Add isort 5 in pre-commit config

peymanslh opened the pull request:
Changes:
- Remove `isort-seed-config` and `isort` 4.
- Upgrade `isort` to version 5 and use the original repository URL.
- Fix some issue in README file

Closes: #24

peymanslh committed:
OPT: Add isort 5 in pre-commit config

Remove isort-seed-config and isort 4

mirzakhany reviewed (approved):
Great! thanks @peymanslh

saeedghx68 commented:
At this moment we don't have a Makefile for the project so I think we should add the isort command to the README.md file

peymanslh commented:
> At this moment we don't have a Makefile for the project so I think we should add the `isort` command to the `README`.md file

I added `isort` config in `pre-commit`, and every time someone wants to commit some changes `pre-commit` runs git hooks and automatically fix imports, I added documentation to how to install git hooks with `pre-commit` in `README` file:

### Install pre-commit git hook
```
pre-commit install
```
And also `pre-commit` handels it's dependencies like `isort` itself.
